perception/multi_object_tracker/src/debugger.cpp

```diff
@@ -108,16 +108,35 @@ void TrackerDebugger::startMeasurementTime(
     processing_time_publisher_->publish<tier4_debug_msgs::msg::Float64Stamped>(
       "debug/input_latency_ms", input_latency_ms);
   }
+  // initialize debug time stamps
+  if (!is_initialized_) {
+    stamp_publish_output_ = now;
+    is_initialized_ = true;
+  }
+}
+
+void TrackerDebugger::endMeasurementTime(const rclcpp::Time & now)
+{
+  stamp_process_end_ = now;
+}
+
+void TrackerDebugger::startPublishTime(const rclcpp::Time & now)
+{
+  stamp_publish_start_ = now;
 }
 
 void TrackerDebugger::endPublishTime(const rclcpp::Time & now, const rclcpp::Time & object_time)
 {
   const auto current_time = now;
-  // pipeline latency: time from sensor measurement to publish
+  // pipeline latency: time from sensor measurement to publish, used for 'checkDelay'
   pipeline_latency_ms_ = (current_time - last_input_stamp_).seconds() * 1e3;
-  if (debug_settings_.publish_processing_time) {
-    // processing time: time between input and publish
-    double processing_time_ms = (current_time - stamp_process_start_).seconds() * 1e3;
+  if (debug_settings_.publish_processing_time && is_initialized_) {
+    // processing latency: time between input and publish
+    double processing_latency_ms = ((current_time - stamp_process_start_).seconds()) * 1e3;
```

Comment:
Since `pipeline_latency_ms` is total latency including processing times and inter-communications, I think the processing_latency is the right wording to indicating total time of interval from input to output.

The `processing_time_ms` of the other packages are measuring time intervals between process start to end.